cve_scripts/cve_scraper.py

```diff
@@ -0,0 +1,47 @@
+import sys, getopt, os
```

Comment:
Just a general note -- this entire script could maybe do with a readme to help people understand what to change if things with the website change in the future. For example, what if the website structure changes and the person looking at this doesn't understand how to simply fix the script so that it can still run on the new folder structure?

Reply:
Okay, will add that to readme